src/Pages/ContentListTable/components/AddContent/AddContent.tsx

```diff
@@ -662,14 +662,14 @@ const AddContent = () => {
                     </FormGroup>
                     <FormGroup fieldId='enable_module_hotfixes'>
                       <Switch
-                        label='Modularity filtering disabled'
-                        labelOff='Modularity filtering enabled'
-                        ouiaId={`module_hotfixes_switch_${moduleHotfixesEnabled ? 'on' : 'off'}`}
+                        label='Modularity filtering enabled'
+                        labelOff='Modularity filtering disabled'
+                        ouiaId={`module_hotfixes_switch_${!moduleHotfixesEnabled ? 'on' : 'off'}`}
                         aria-label='enable_module_hotfixes'
                         hasCheckIcon
                         id={'module-hotfixes-switch' + index}
                         name='module-hotfixes-switch'
-                        isChecked={moduleHotfixesEnabled}
+                        isChecked={!moduleHotfixesEnabled}
                         onChange={() => {
                           updateVariable(index, { moduleHotfixesEnabled: !moduleHotfixesEnabled });
```

Comment:
This is a lot of double negatives, for my 🧠 's sanity would you mind switching the name and reversing the defaults?  😆

Reply:
updated. let me know if that's any better